config/locales/devise.en.yml

```diff
@@ -75,7 +75,7 @@ en:
       last_name: "Last Name:*"
       first_name: "First Name:*"
       company: "Company:"
-      email: "Email:"
+      email: "Email:*"
```

Comment:
This might be out of scope, but we can use the class "required" on labels to add "*"s to fields instead of placing them in the YML. We do it in several other places currently. You can call the javascript method "set_required_fields()" to do this.